packages/css/src/index.ts

```diff
@@ -237,5 +238,5 @@ export const css = (args: SystemStyleObject = {}) => (
     }
   }
 
-  return result
+  return result as SerializedStyles
```

Comment:
We probably don't want to do this..but I couldn't figure out how to get this function to play nicely with the styles prop in the `Global` component (`@emotion/core`).

Emotion's css type definition: 
https://github.com/emotion-js/emotion/blob/2f608a2e038d9a3c21dc506fc19c6786b24a4bec/packages/css/types/index.d.ts#L19

Would love some suggestions 🙏

Reply:
I don't have a local copy to play around with but I noticed `SerializedStyles` is originally from [@emotion/utils](https://github.com/emotion-js/emotion/blob/master/packages/utils/types/index.d.ts#L29-L34) and looks like:

```typescript
export interface SerializedStyles {
  name: string
  styles: string
  map?: string
  next?: SerializedStyles
}
```

so it looks like the initial `result` object on L204 would need `name` and `styles` properties?

Reply:
Yeah it's incorrect. Thanks for pointing that out. I looked at what `result` contains and it's a css object.
I found a better type to use here and it keeps the `Global` component happy